README.md

```diff
@@ -93,51 +93,63 @@ jobs:
           publish-profile: ${{ secrets.AZURE_WEB_APP_PUBLISH_PROFILE }}
 ```
 
-The following variable should be replaced in your workflow:
+The following variables should be replaced in your workflow:
 
 - `<PLATFORM_NAME>`
-    - Programming platform used by the web app that's being deployed, **optional**.
+  - Programming platform used by the web app that's being deployed
+  - **Optional** - Oryx will detect the provided application's platform if not provided
 
 - `<PLATFORM_VERSION>`
-    - Version of programming platform used by the web app, **optional**.
-    - Oryx will determine the version from source files if version was not set.
-    - Default version of each platform if version was not set and Oryx couldn't determine version :
-      - .NET Core 3.1
-      - Node 12.16
-      - PHP 7.3
-      - Python 3.8
+  - Version of programming platform used by the web app
+  - **Optional** - Oryx will determine the version from source files if version was not set.
+  - The following are the default versions used per platform if a version cannot be detected by Oryx:
+    - Golang 1.18

```

Comment:
Please consider redirecting to the Oryx repo for this rather than hardcoding these here. Else every time we're updating these we'll need to remind to update these for little value overall.